src/clr-angular/utils/focus/focus-on-view-init/focus-on-view-init.module.ts

```diff
@@ -6,11 +6,14 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 
+import { FOCUS_ON_VIEW_INIT_PROVIDER } from './focus-on-view-init.provider';
+
 import { FOCUS_ON_VIEW_INIT_DIRECTIVES } from './index';
```

Comment:
We don't want to define anything in the index file, only reexport. That was one thing that caused some problems in the past, can you move those into here?

Reply:
We can do that. But several of our components and utility directives are using this pattern.

Reply:
Anything that we publicly export into public_api does not have that pattern, there are some old ones in utilities we don't export that weren't refactored and they are ok.

Reply:
Ah I see. I made these changes looking at the focus-trap directive. So that makes sense.